run/multibit2john.py

```diff
@@ -1344,22 +1344,18 @@ def process_file(filename):
                         return
             return
 
-    version = 1  # MultiBit Classic
     pdata = b"".join(data.split())
     if len(pdata) < 64:
         sys.stderr.write("%s: Short length for a MultiBit wallet file!\n" % bname)
         return
+    pdata = base64.b64decode(pdata[:64])
 
-    try:
-        pdata = base64.b64decode(pdata[:64])
-        if not pdata.startswith("Salted__"):
-            version = 2
-        if len(pdata) < 48:
-            # sys.stderr.write("%s: Short length for a MultiBit wallet file!\n" % bname)
-            # return
-            version = 2  # MultiBit HD possibly?
-    except:
+    if pdata.startswith(b"Salted__"):
```

Comment:
The `"Salted__"` vs `b"Salted__"` is a Python 2/3 thing, right? Does the latter syntax work with both?

Reply:
IIRC `b"Salted__"` works with both the major Python versions.